server/database.sql

```diff
@@ -0,0 +1,33 @@
+CREATE DATABASE blogapp;
```

Comment:
This here is going to work for the first time, but when you decide to alter some table it will raise SQL exception, database already exists or something. Always create db manually, and tables via migrations :).

Reply:
I created db manually. I just added this code to see what i did in db.

Reply:
Try altering tables and run it again and see what will pop up. Crash...